nbd/nbd-package/DEBIAN/preinst

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+CURVETAB=/etc/curve/curvetab
+CURVETABBAK=/etc/curve/curvetab-bak
+
+if [ -f ${CURVETAB} ]; then
+    cp -f ${CURVETAB} ${CURVETABBAK}
```

Comment:
Add '-p' cp command option to preserve 'mode,ownership,timestamps' ?

Reply:
Don’t you need to copy the contents of the existed curvetab ?

Reply:
fix